### ISSUE TITLE
Clean theme should describe thumbnail article images on homepage

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4213]
The article images were ignored in the screenreader testing, but are they truly decorative?

This relates to #2825 implementation. But currently the article images are not focusable and so completely ignored.   These images are arguably not decorative.  Please consider providing alt-text functionality instead.

example https://publications.essex.ac.uk/. 
<img width="980" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/87f75845-2ae6-451b-b0b6-3aa24ed76149">

**Comment by StephDriver** [comment]
from #4194 
- Sample 5.3, item 14. article images are ignored as decorative - are they decorative?

**Comment by StephDriver** [comment]
This is user-content, which cannot be addressed until the new alt-text functionality is in place, from https://github.com/BirkbeckCTP/janeway/issues/2825

**Comment by ajrbyers** [comment]
Alt Text feature has been merged. Closing.
